api/api/src/main/scala/uk/ac/wellcome/platform/api/services/FiltersAndAggregationsBuilder.scala

```diff
@@ -87,6 +87,7 @@ class FiltersAndAggregationsBuilder(
     case _: GenreFilter            => Some(AggregationRequest.Genre)
     case _: SubjectFilter          => Some(AggregationRequest.Subject)
     case _: LicenseFilter          => Some(AggregationRequest.License)
+    case _: CollectionDepthFilter  => None
```

Comment:
Maybe this pattern matching could just have a single case for all patterns returning `None`, rather than having to define them all individually?

```
case _ => None
```

Reply:
I quite like having it there as it errors if you don't, and make you remember to think about it.